src/components/templates/Camp/components/Map.tsx

```diff
@@ -64,18 +69,14 @@ export const Map: React.FC<PropsType> = ({
               }}
               key={room.title}
               onClick={() => {
-                if (IS_BURN) {
-                  setSelectedRoom(room);
-                  setIsRoomModalOpen(true);
+                if (!IS_BURN) {
+                  openModal(room);
                 } else {
                   setRoomClicked((prevRoomClicked) =>
                     prevRoomClicked === room.title ? undefined : room.title
                   );
                 }
               }}
-              onMouseEnter={() => {
```

Comment:
You still need this for burn mode, you able to keep it?

Reply:
reverted that back in